resources/shaders/dxr_reflection_main.hlsl

```diff
@@ -96,10 +98,14 @@ void ReflectionRaygenEntry()
 	{
 		// A value of 1 in the output buffer, means that there is shadow
 		// So, the far plane pixels are set to 0
-		output_refl_shadow[DispatchRaysIndex().xy] = float4(0, 0, 0, 0);
+		output_reflection[screen_co] = float4(0, 0, 0, 0);
+		output_shadow[screen_co] = 0;
+		output_dirT_buffer[screen_co] = float4(0, 0, 0, 0);
```

Comment:
style guide

Reply:
What do you mean with style guide?

Reply:
output_dirT_buffer. dirT Is camel cased.

Reply:
Changed it. Also changed it in the other shader files.